scss/_reboot.scss

```diff
@@ -163,12 +161,10 @@ dfn {
   font-style: italic; // Add the correct font style in Android 4.3-
 }
 
-// stylelint-disable font-weight-notation
```

Comment:
There was no need for this (see https://github.com/stylelint/stylelint/tree/master/lib/rules/font-weight-notation#font-weight-notation)